src/pages/error/Error.tsx

```
@@ -0,0 +1,29 @@
+import { IcError } from '@components/icons';
+import Button from '@components/button/Button';
+import { useNavigate } from 'react-router-dom';
+import { routePath } from '@routes/routePath';
+
+import * as styles from './error.css';
+
+const Error = () => {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)